From: Caleb Connolly <caleb.connolly@linaro.org>
Date: Fri, 17 May 2024 18:12:29 +0200
Subject: [PATCH] sdm845.config: enable vbus regulator

Needed for display bringup on sm8250

Signed-off-by: Caleb Connolly <caleb.connolly@linaro.org>
(cherry picked from commit 389bb187aff2cdeaa5205b741b0433964722623e)
Link: https://gitlab.com/sdm845-mainline/linux
Signed-off-by: Richard Acayan <mailingradian@gmail.com>
---
 arch/arm64/configs/sdm845.config | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/sdm845.config b/arch/arm64/configs/sdm845.config
index e102374..21ef30d 100644
--- a/arch/arm64/configs/sdm845.config
+++ b/arch/arm64/configs/sdm845.config
@@ -151,6 +151,8 @@ CONFIG_PSTORE_DEFLATE_COMPRESS=y
 CONFIG_USB_CONFIGFS=y
 CONFIG_USB_CONFIGFS_F_HID=y
 
+CONFIG_REGULATOR_QCOM_USB_VBUS=m
+
 CONFIG_LEDS_TRIGGER_ONESHOT=y
 CONFIG_LEDS_TRIGGER_BACKLIGHT=y
 CONFIG_LEDS_TRIGGER_ACTIVITY=y
